CMakeLists.txt

```diff
@@ -378,6 +378,23 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake/Modules")
 find_package(Boost 1.70.0 REQUIRED COMPONENTS filesystem log log_setup thread
                                               program_options system)
 
+# Diskhash
```

Comment:
About to change these few lines to actually use cmake for building the library -- current form was just easier for testing.